bigquery_etl/parse_udf.py

```diff
@@ -64,17 +67,21 @@ def from_raw(raw_udf, full_sql):
         return ParsedUdf(*astuple(raw_udf), full_sql)
 
 
-def read_udf_dir(d):
-    """Read contents of d into RawUdf instances."""
-    for root, dirs, files in os.walk(d):
-        for filename in files:
-            if not filename.startswith("."):
-                yield RawUdf.from_file(os.path.join(root, filename))
+def read_udf_dirs(*udf_dirs):
+    """Read contents of udf_dirs into dict of RawUdf instances."""
```

Comment:
we were converting this to a dict everywhere we used it, so I replaced the function instead of just making a new one